include/SDL_config.h.in

```diff
@@ -1,491 +1,1195 @@
-/*
```

Comment:
This is broken, I guess this is a result of `autoreconf` or `autoheader`: don't use.
Changes to SDL_config.h.in should be as simple as the following:
```diff
--- a/include/SDL_config.h.in
+++ b/include/SDL_config.h.in
@@ -208,4 +208,7 @@
 #undef HAVE_ELF_AUX_INFO
 #undef HAVE_POLL
+#undef HAVE_MEMFD_CREATE
+#undef HAVE_MKOSTEMP
+#undef HAVE_POSIX_FALLOCATE
 #undef HAVE__EXIT
 
```